metricflow/specs/specs.py

```diff
@@ -292,6 +295,67 @@ def accept(self, visitor: InstanceSpecVisitor[VisitorOutputT]) -> VisitorOutputT
         return visitor.visit_dimension_spec(self)
 
 
+class TimeDimensionSpecField(Enum):
+    """Fields of the time dimension spec.
+
+    The value corresponds to the name of the field in the dataclass. This should contain all fields, but implementation
```

Comment:
We might consider switching to `dataclasses.fields` or some other utility in the `dataclasses` module for this. Depends on what we're trying to do, I guess.

Reply:
It's a little unclear how those would be used since those are dynamic, and I was looking for something specific I can reference in code.